test/handler.test.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
```

Comment:
Suggestion: Call this file "index.test.js" so it's easier to be associated with "index.js" just by looking at fileName